static/js/popovers.js

```diff
@@ -509,6 +512,14 @@ export function toggle_actions_popover(element, id) {
 
         const should_display_delete_option =
             message_edit.get_deletability(message) && not_spectator;
+
+        const should_display_star_message =
+            !message.locally_echoed &&
+            not_spectator &&
+            window.innerWidth < media_breakpoints_num.sm;
+        const should_display_add_reaction_option =
+            message.sent_by_me || window.innerWidth < media_breakpoints_num.sm;
```

Comment:
does this check need a `not_specator`? `message.sent_by_me` was always false for spectators, but this new value isn't.

Reply:
Yeah, fixed.